src/main/java/com/payments_jdbc/JdbcConection.java

```diff
@@ -0,0 +1,157 @@
+
+	}
+
+	public static void AccountDetailsInAsscendingOrder() throws SQLException {
+		Connection connect = getConnection();
+
+		String query = "select * from Payments,Accounts_table order by name;";
+		PreparedStatement preparedStatement = connect.prepareStatement(query);
+		ResultSet resultSet = preparedStatement.executeQuery();
+
+		while (resultSet.next()) {
+			System.out.println(
+					"--------------------------------------------------------------------------------------------------");
+			System.out.printf("|%-30s|%-30s|%-30s|%n", resultSet.getString("Name"), resultSet.getInt("age"),
+					resultSet.getString("AccountId"));
+		}
+		System.out.println(
+				"--------------------------------------------------------------------------------------------------");
+
+		System.out.println("sucessfully names sorted in asscending order");
+
+	}
+
+	public static void printRecordsInCurrentWeek() throws SQLException {
+
+		Connection connect = getConnection();
+		String query = "SELECT * FROM accounts_table WHERE WEEK(dateOfJoining) = WEEK(CURDATE())";
+
+		PreparedStatement statement = connect.prepareStatement(query);
+		ResultSet resultSet = statement.executeQuery();
+		System.out.println("Retrieval successful");
+
+		while (resultSet.next()) {
+			System.out.println(
+					"---------------------------------------------------------------------------------------------------------------------");
+			System.out.printf("|%-30s|%-30s|%-30s|%-30s|%n", resultSet.getDate("dateOfJoining"),
+					resultSet.getInt("age"), resultSet.getString("gender"), resultSet.getString("name"));
+		}
+		System.out.println(
+				"--------------------------------------------------------------------------------------------------------------------------");
+	}
+
+	public static void AccountBalance(String Accountid) throws SQLException {
+		Connection connect = getConnection();
+		String query = "SELECT a.accountId,a.name, a.openingbalance, p.purposeOfPayment, p.amountPaid, \r\n"
+				+ "       (a.openingbalance - p.amountPaid) AS Account_Balance \r\n" + "FROM accounts_table AS a \r\n"
+				+ "JOIN payments AS p \r\n" + "ON a.accountId = p.accountId\r\n" + "WHERE a.accountId = ?"
+				+ "LIMIT 0, 1000;";
+		PreparedStatement statement = connect.prepareStatement(query);
+		statement.setString(1, Accountid);
+		ResultSet resultSet = statement.executeQuery();
+		System.out.println("Retrieval successful");
+
+		while (resultSet.next()) {
+			System.out.println(
+					"--------------------------------------------------------------------------------------------------");
+			System.out.printf("|%-30s|%-30s|%-30s|%n", resultSet.getLong("accountid"), resultSet.getString("name"),
+					resultSet.getString("Account_Balance"));
+		}
+		System.out.println(
+				"--------------------------------------------------------------------------------------------------");
+
+	}
+
+	public static void deleteAccount(long id) throws SQLException {
```

Comment:
what if I gave random accountId ??

Reply:
solved this issue by using matches method